test/integration_test/data/headers_cl.json

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
short test data files are preferable to long ones. If you can reduce the number of entries in this file and still have the test coverage, that would be a good thing.